data/yam/agama/auto/lib/scripts.libsonnet

```diff
@@ -0,0 +1,43 @@
+local enable_root_login = {
+  name: 'enable root login',
+  chroot: true,
+  body: |||
+    #!/usr/bin/env bash
+    echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
+  |||
+};
+
+{
+  default: {
+    post: [ enable_root_login ]
+  },
+  multipath: {
+    pre: [{
+      name: 'activate multipath',
+      body: |||
+        #!/bin/bash
+        if ! systemctl status multpathd ; then
+          echo 'Activating multipath'
+          systemctl start multipathd.socket
+          systemctl start multipathd
+        fi
+      |||,
+    }]
+  },
+  wipefs: {
+    pre: [
+      {
+        name: 'wipefs',
+        body: |||
+          #!/usr/bin/env bash
+          for i in `lsblk -n -l -o NAME -d -e 7,11,254`
+              do wipefs -af /dev/$i
+              sleep 1
+              sync
+          done
+        |||
+      }
+    ],
+    post: [ enable_root_login ],
```

Comment:
we should not make wipefs (let's use full names, wipe_filesystem) dependent on enable_root_login, we should compose them, add them seprately.